### PR TITLE
[NORMAL 24-03-2024] UI for adding expense

### PR DESCRIPTION
**Pull request by dorji-dev**
- Calendar UI
- Select UI
- Textarea UI
- Date picker single UI
- Add expense UI

### CODE REVIEW

File: src/components/add-new-expense/add-new-expense-form.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          className="mt-[8px] font-normal"
+        />
+      </label>
+      <Button type="submit" className="w-full">
+        Save
+      </Button>
+    </form>
+  );
+};
+
```
Comment (dorji-dev) [inline src/components/add-new-expense/add-new-expense-form.tsx:64]: Screenshot 2024-03-22 at 00 46 44

File: src/components/ui/calendar.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          <MdKeyboardArrowRight className="h-[16px] w-[16px]" />
+        ),
+      }}
+      {...props}
+    />
+  );
+}
+Calendar.displayName = "Calendar";
+
+export { Calendar };
```
Comment (dorji-dev) [inline src/components/ui/calendar.tsx:111]: Screenshot 2024-03-22 at 00 49 20

File: src/components/ui/date-picker-single.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            fromYear={1960}
+            toYear={2030}
+            onSelect={onDateSelect}
+            {...props}
+          />
+        </PopoverContent>
+      </Popover>
+    </div>
+  );
+}
```
Comment (dorji-dev) [inline src/components/ui/date-picker-single.tsx:70]: Screenshot 2024-03-22 at 00 49 44

File: src/components/ui/select.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  SelectGroup,
+  SelectValue,
+  SelectTrigger,
+  SelectContent,
+  SelectLabel,
+  SelectItem,
+  SelectSeparator,
+  SelectScrollUpButton,
+  SelectScrollDownButton,
+};
```
Comment (dorji-dev) [inline src/components/ui/select.tsx:161]: Screenshot 2024-03-22 at 00 51 01

File: src/components/add-new-expense/add-new-expense-form.tsx (hunk 1 above)
Comment (Sonam-Choden) [reply src/components/add-new-expense/add-new-expense-form.tsx:64]: 👍

**Review by Sonam-Choden — APPROVED** [review]

File: src/components/add-new-expense/add-new-expense-form.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import { Textarea } from "../ui/textarea";
+import { Button } from "../ui/button";
+
+const AddNewExpenseForm = () => {
+  const [date, setDate] = useState<Date>();
+
+  return (
+    <form className="space-y-[20px]">
+      <div>
+        <span className="block font-medium mb-[8px]">Date</span>
```
Comment (Chojay2) [inline src/components/add-new-expense/add-new-expense-form.tsx:22]: use P tag, will not need to give display block property

**Review by karmatenzi — APPROVED** [review]

**Pull request merged**